cgsmiles/resolve.py

```diff
@@ -185,6 +184,51 @@ def edges_from_bonding_descrpt(self):
                 order = 1
             self.molecule.add_edge(edge[0], edge[1], bonding=bonding, order=order)
 
+    def squash_atoms(self):
+        """
+        Applies the squash operator.
+        """
+        bondings = nx.get_edge_attributes(self.molecule, 'bonding')
+        squashed = False
+        for edge, bonding in bondings.items():
+            # we have a squash operator
+            if bonding[0].startswith('!'):
+                # find all hydrogens to remove
+                # and which atoms to connect
+                nodes_to_remove = [edge[1]]
+                new_edge_nodes = []
+                for hnode in self.molecule.neighbors(edge[1]):
+                    if self.molecule.nodes[hnode].get('element', 'Nan') == 'H':
```

Comment:
Nan element? Why not None, or '*' (the smiles unknown atom standard)

Reply:
any works fine; only if it is a CG graph it does not have an element attribute